### PR TITLE
BE & view checkout

### PR DESCRIPTION
**Pull request by khieutrung12**
## Related Tickets:

-   [#45921](https://edu-redmine.sun-asterisk.vn/projects/ecommerce-intern-hn/issues/45921)

## WHAT this PR does?

BE & view checkout

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

**Comment by khieutrung12** [comment]
ready

**Comment by khieutrung12** [comment]
ready

**Review by DTHNhung — APPROVED** [review]

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(StoreRequest $request)
+    {
+        if (Session::has('cart')) {
+            $order_status = config('app.startOrderStatus');
+            $code = Str::random(6);
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: bỏ số vào config nha

**Comment by khieutrung12** [comment]
ready

**Pull request merged**